### DISCUSSION TITLE
Jira-assistant Worklog Report "Add Jira Group" does not work

### DISCUSSION THREAD

**Discussion by NadaReska** [shridhar-tl/jira-assistant#354] [Q&A]
When trying to add a jira group to "User Group" I can see suggestions. However, when I press the tic, nothing happens. 
![image](https://github.com/shridhar-tl/jira-assistant/assets/168548089/e75e0126-2ed2-496d-a954-8db49085ad83)

**Comment by shridhar-tl** [comment]
That depends on if you have permission to pull the Jira Group or not. Either you should be an admin in Jira or you should be explicitly provided access to access the Jira group.
